maquetado/scss/titles.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  font-size: 30px;
```

Comment:
You can create the`fonts.scss` file with `$text-big: 24px;`, `$text-medium: 16px` and `$text-small: 12px;` sass variables and use these variables here.